sonar-plugin-api/src/main/java/org/sonar/api/ce/measure/test/TestIssue.java

```diff
@@ -78,16 +78,27 @@ public String severity() {
     return severity;
   }
 
+  /**
+   * @deprecated since 5.5, replaced by {@link #effort}
+   */
   @Override
   @CheckForNull
+  @Deprecated
   public Duration debt() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)